content/about/introduction/introduction.html

```diff
@@ -14,166 +14,161 @@
-          Consequently, some developers may find it necessary to go beyond  simply combining various APG patterns to create an experience that supports their UI requirements.
-          If building UI that deviates from APG patterns, ensure the design and behavior of the UI:
+          しかし、APGパターンライブラリは、アクセシブルなユーザーインターフェースを構築するためのすべての可能で、有効で、おそらく使用可能な方法のカタログとして機能することを意図しているわけではありません。
+          そのため、一部の開発者は、UI要件をサポートする体験を作成するために、単にさまざまなAPGパターンを組み合わせることを超えて進む必要があると感じるかもしれません。
+          APGパターンから逸脱するUIを構築する場合は、UIのデザインと動作が以下と一致していることを確認してください：
         </p>
         <ul>
-          <li>Are consistent with relevant guidance covered by <a href="../../practices/practices.html">APG practices</a>.</li>
-          <li>Comply with normative requirements of the <a class="specref" href="#abstract">ARIA specification</a>.</li>
-          <li>Follow or mimic keyboard conventions in similar APG patterns or native operating system widgets.</li>
+          <li><a href="../../practices/practices.html">APG実践</a>によってカバーされている関連するガイダンスと一致している。</li>
+          <li><a class="specref" href="#abstract">ARIA仕様</a>の規範的な要件に準拠している。</li>
+          <li>同様のAPGパターンまたはネイティブのオペレーティングシステムウィジェットのキーボード規約に従う、またはそれを模倣する。</li>
         </ul>
       </section>
 
       <section id="not-design-system">
-        <h2>APG is not a UI Design System</h2>
+        <h2>APGはUIデザインシステムではありません</h2>
         <p>
-          While the APG includes a library of patterns and functional examples, it's objectives do not include providing a comprehensive design system or production-ready code.
-          Not only would developing a production-ready design system be beyond the capacity of the Authoring Practices Task Force, developing examples that satisfy requirements of a unified design system would inhibit its primary aim of building broad understanding of relevant practices.
-          APG examples, including  code style, are designed to facilitate learning the accessibility concepts they illustrate.
-          To show how different approaches to design can satisfy accessibility requirements, the styles and behaviors of examples are sometimes intentionally inconsistent from one another.
-          Additionally, the examples are intended to be reasonably realistic and utilize modern coding techniques, but their purpose is not to teach other dimensions of web engineering, e.g., localization and robust platform interoperability.
-          Further information about platform and browser support is conveyed in the
-          <a href="../../practices/read-me-first/read-me-first-practice.html">Read Me First</a>.
+          APGにはパターンと機能例のライブラリが含まれていますが、包括的なデザインシステムや本番環境に適したコードを提供することは目的に含まれていません。
+          本番環境に適したデザインシステムを開発することは、オーサリングプラクティスタスクフォースの能力を超えるだけでなく、統一されたデザインシステムの要件を満たす例を開発することが、その主な目的である関連する実践の理解を構築することを妨げるでしょう。
```

Comment:
broadを訳出しつつ、少し流れが自然になるように助詞を変更しました

```suggestion
          本番環境に適したデザインシステムを開発することは、オーサリングプラクティスタスクフォースの能力を超えるだけでなく、統一されたデザインシステムの要件を満たす例を開発することは、その主な目的である関連する実践の幅広い理解を構築することを妨げるでしょう。
```